Source/cmMakefileExecutableTargetGenerator.cxx

```diff
@@ -136,7 +136,12 @@ void cmMakefileExecutableTargetGenerator::WriteNvidiaDeviceExecutableRule(
 
   // Build list of dependencies.
   std::vector<std::string> depends;
+
+  // Build other list of dependencies, which will use to get json's links command
```

Comment:
Please, include a small description of what is the "other" list, how is it different to previous

Reply:
which will **be used** to get json's link command**s**